.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test: --platform vm
+  # No parallelism on Firefox (-j 1)
+  # Causes flakiness – need to investigate
+  - test: --platform firefox -j 1
+  - dartanalyzer
```

Comment:
I'd put this under `matrix.include` as well. I don't think there's likely to be a lot of value in double-testing analysis.

Reply:
As long as stable is supported, I think it's good to track – help find potential API changes that are not backwards compatible.

Reply:
We already get that from testing on stable. This package isn't doing anything statically complex enough to be worth the extra job time from this task.

Reply:
ack